src/main/java/com/opencart/pages/AdminNavigation.java

```diff
@@ -0,0 +1,34 @@
+package com.opencart.pages;
+
+import lombok.Getter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+@Getter
+public class AdminNavigation extends BasePage {
+
+    @FindBy(id = "menu-system")
+    private WebElement systemMenu;
+
+    @FindBy(xpath = ".//ul[@id = 'collapse7']//a[contains(@href,'#collapse7-2')]")
```

Comment:
I believe it is better to update this locator, looks like that id is autogenerated, you can search by text here